### ISSUE TITLE
Audio reference for input

### ISSUE THREAD

**Issue by hufube** [neonbjb/tortoise-tts#63]
First, congratulations for the great job you have done, really a very professional result. On the other hand, I would like to ask you if the entry of reference audios is plausible, to give connotation to a phrase through, say, the spectrogram of another.

It would be very convenient for the intonation of certain words, as is the addition of the ARPA format for the text, although I suppose that would be more complicated and would require another training. Anyway, I congratulate you for your effort.

**Comment by neonbjb** [comment]
No, this model doesn't do that. Agreed that increased control over phonetics would also be very helpful and it's something I'm looking at. #17
